internal/characters/travelerdendro/asc.go

```diff
@@ -0,0 +1,66 @@
+package travelerdendro
+
+import (
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/player/character"
+	"github.com/genshinsim/gcsim/pkg/modifier"
+)
+
+// Gets removed on swap - from Kolbiri
+func (c *char) a1Init() {
+	c.Core.Events.Subscribe(event.OnCharacterSwap, func(args ...interface{}) bool {
+		prev := args[0].(int)
+		prevChar := c.Core.Player.ByIndex(prev)
+		prevChar.DeleteStatMod("dmc-a1")
```

Comment:
either you rename all event sub names/status names/logs/... to using dmc or you change them to all use travelerdendro. I prefer dmc. Also make them all lowercase pls

Reply:
I've updated the ones that say travelerdendro to be dmc